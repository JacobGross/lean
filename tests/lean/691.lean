theorem foo : Type₁ := unit

example : foo = unit :=
by unfold foo

example : foo = unit :=
by unfold [foo]

example : foo = unit :=
by rewrite [↑foo]

example : foo = unit :=
by rewrite [↑[foo] ]
