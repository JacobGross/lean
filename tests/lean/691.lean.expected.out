691.lean:4:10: error: invalid 'rewrite' tactic, cannot unfold 'foo' which is still in the theorem queue. Use command 'reveal foo' to access its definition.
691.lean:7:11: error: invalid 'rewrite' tactic, cannot unfold 'foo' which is still in the theorem queue. Use command 'reveal foo' to access its definition.
691.lean:10:13: error: invalid 'rewrite' tactic, cannot unfold 'foo' which is still in the theorem queue. Use command 'reveal foo' to access its definition.
691.lean:13:14: error: invalid 'rewrite' tactic, cannot unfold 'foo' which is still in the theorem queue. Use command 'reveal foo' to access its definition.
